src/Http/Controllers/Views/KpiController.php

```diff
@@ -18,8 +18,9 @@ class KpiController extends Controller
      */
     public function show($idx = 6)
     {
-        // If index is not in range, set it on index of last month
-        $idx = $idx == 0 || $idx > 6 ? 6 : $idx;
+        if($idx == 0 || $idx > 6) {
+            abort(404);
+        }
 
         $date = Carbon::now()->subMonths(6 - $idx);
```

Comment:
The month selection doesn't work quite yet. Today is April and if I select "Feb", I'm redirected to `admin/kpis/5`. This gives `subMonths(6 - 5)` which returns March instead of February.

When I visit the page in April I want to see the KPIs of March (`subMonths(1)`) and for February it should call `subMonths(2)`.